docs/platforms/apple/common/index.mdx

```diff
@@ -1,34 +1,172 @@
-<PlatformContent includePath="getting-started-primer" />
+On this page, we get you up and running with Sentry's Apple SDK, automatically reporting errors and exceptions in your application.
 
-On this page, we get you up and running with Sentry's SDK.
+<Note>
+
+If you don't already have an account and Sentry project established, head over to [sentry.io](https://sentry.io/signup/), then return to this page.
+
+</Note>
+
+<PlatformSection supported={["apple.visionos"]}>
+
+<Alert level="info" title="Experimental Support">
 
-Don't already have an account and Sentry project established? Head over to [sentry.io](https://sentry.io/signup/), then return to this page.
+The support for [visionOS](https://developer.apple.com/visionos/) is currently experimental.
+Experimental support means it may have bugs. We recognize the irony.
+
+</Alert>
+
+</PlatformSection>
 
 ## Install
 
-Sentry captures data by using an SDK within your application’s runtime.
+<OnboardingOptionButtons
+  options={[
+    'error-monitoring',
+    'performance',
+    'profiling',
+  ]}
+/>
+
+Sentry captures data by using an SDK within your application's runtime. These are platform-specific and allow Sentry to have a deep understanding of how your application works.
+
+<PlatformSection notSupported={["apple.ios"]}>
 
-<PlatformContent includePath="getting-started-install" />
+We recommend installing the SDK with Swift Package Manager (SPM), but we also support alternate [installation methods](install/). To integrate Sentry into your Xcode project, open your App in Xcode and open **File > Add Packages**. Then add the SDK by entering the git repo url in the top right search field:
+
+```
+https://github.com/getsentry/sentry-cocoa.git
+```
+
+</PlatformSection>
+
+<PlatformSection supported={["apple.ios"]}>
+We recommend installing the SDK through our [Sentry Wizard](https://github.com/getsentry/sentry-wizard) by running the following command inside your project directory:
+
+```bash
+brew install getsentry/tools/sentry-wizard && sentry-wizard -i ios
+```
+
+This will patch your project and configure the SDK. You only need to patch the project once, then you can add the patched files to your version control system.
+If you prefer, you can also [set up the SDK manually](/platforms/react-native/manual-setup/manual-setup/) or follow the instructions below to adapt the [configuration](#configure).
```

Comment:
```suggestion
This will patch your project and configure the SDK. You'll only need to patch the project once, then you'll be able to add the patched files to your version control system.
If you prefer, you can also [set up the SDK manually](/platforms/react-native/manual-setup/manual-setup/) or follow the instructions below to adapt the [configuration](#configure).
```